### ISSUE TITLE
Is there an "Upgrade" path if I already have nflfastR?

### ISSUE THREAD

**Issue by AccidentalGuru** [nflverse/nflverse#6]
I don't want to make a mess because I just got everything tidy.

Also, I think the nflreadR package is inconsistent as it is nflreadr in CRAN.

**Comment by mrcaseb** [comment]
I am not sure how to understand this question. Are you currently running nflfastR only and would like to install nflverse?
This is most likely completely safe. However, if you are running a very old version of nflfastR or dplyr you may have to update them first.

> Also, I think the nflreadR package is inconsistent as it is nflreadr in CRAN.

I am confused about this. Where did you find `nflreadR` in this package? Did you eventually misread `nflseedR`?

**Comment by AccidentalGuru** [comment]
Sorry for any confusion.
First half you nailed it! 
I had been running nflfastR and got it hooked up to Postgres, then saw nflreadr was available and installed it, but I removed them and installed nflverse.

I did not see an `nflreadR`, but tried and failed to install one, and thought others might do the same since the other packages are capital "R" endings (e.g. `nflreadr` is the only lower case trailing "r" as opposed to `nflscrapR`, `nflfastR`, `nflseedR`, `espnscrapR`).

It is likely trivial if `nflverse` will cover all the packages going forward. 

Thanks for all your hard work.
